packages/snaps-sdk/src/types/methods/request-device.ts

```diff
@@ -0,0 +1,45 @@
+import { number, object, optional } from '@metamask/superstruct';
+
+import type { Describe } from '../../internals';
+import type { Device, DeviceType } from '../device';
+
+export type DeviceFilter = {
+  /**
+   * The vendor ID of the device.
+   */
+  vendorId?: number;
```

Comment:
Do we just expand this in the future for other devices in one big object?

Reply:
If there's some type-specific filters later, we could do a discriminated union type, with filters specific to each type.

```ts
{
  type: 'some-new-type',
  filters: FiltersForSomeNewType[];
}
```